lib/nomadsl.rb

```diff
@@ -544,8 +544,12 @@ def package(id)
 
   def _vault_aws_creds(path, export)
     prefix = export ? "export " : ""
+    path = path.is_a?(String) ? [path] : path
+    args = path.reduce("") do |concat, str|
+      concat = "#{concat} \"#{str}\""
+    end
```

Comment:
I might be more aggressive and do:

    args = [path].flatten.map do |s|
      %Q("#{s}")
    end.join(" ")

But the end state is no different really. There's a risk of quote escape failure, but that was already there, and seems unlikely to come up anyway, although it's probably worth fixing at some point.